fe-games/src/components/ReviewsList.jsx

```diff
@@ -0,0 +1,39 @@
+import { useState, useEffect } from "react";
+import { fetchReviews } from "./api";
+
+export function ReviewsList() {
+  const [currentReviews, setCurrentReviews] = useState([]);
+
+  useEffect(() => {
+    fetchReviews()
+      .then(({ reviews }) => {
+        console.log();
+        return reviews;
```

Comment:
Please remove any stray console logs 👀. Do you need 2 .then blocks in your useEffect here 🤔